src/renderer/helpers/assetHelper.ts

```diff
@@ -83,7 +80,7 @@ export const CACAO_DECIMAL = 10
  * 1 RUNE == 100,000,000 ð (tor)
  * 0.00000001 RUNE == 1 ð (tor)
  * */
-export const BNB_DECIMAL = 8
+export const BTC_DECIMAL = 8
```

Comment:
we don't need this.

Reply:
Fixed but this ideally should be in the package.